src/models/article_models.js

```diff
@@ -0,0 +1,154 @@
+import createModel from './base_model';
+
+/**
+ * @class Class for Brand model.
+ * @param {String} name - name of brand.
+ * @constructor
+ */
+const Brand = createModel({
```

Comment:
Now Brand, Price, Partner etc.. are in 2 locations?

catalog_api_models and article_models